src/Integration/Handler.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Sentry\Integration;
+
+/**
+ * This class handles the state of already installed integrations.
+ * It makes sure to call {@link IntegrationInterface::setupOnce} only once per integration.
+ *
+ * Class Handler
+ */
+final class Handler
+{
+    private static $integrations = [];
+
+    /**
+     * Calls {@link IntegrationInterface::setupOnce} for all passed integrations if it hasn't been called yet.
+     *
+     * @param array $integrations
+     *
+     * @return array
```

Comment:
We can describe the return type in a better way than a generic array: `array<string, IntegrationInterface>`